### PR TITLE
Update trajectory.js docs

### PR DESCRIPTION
**Pull request by squirrelo**

### CODE REVIEW

File: emperor/support_files/js/trajectory.js [hunk 1]
```diff
@@ -60,20 +48,39 @@ define([
     // the first three axes of the data points
     this.coordinates = coordinates;
 
-    // minimum distance in the gradient
+    /**
+     *minimum differential between samples in the trajectory; the value is
```
Comment (ElDeveloper) [inline emperor/support_files/js/trajectory.js]: Can you capitalize this line and add a leading space to this line?

File: emperor/support_files/js/trajectory.js [hunk 2]
```diff
@@ -60,20 +48,39 @@ define([
...
+     * computed using the gradientPoints array.
+     * @type {float}
+     */
     this.minimumDelta = minimumDelta;
 
     // this value determines how fast the animation will run for now let's use
     // 5 and stick to it as a good default value; 60 was way too slow
+    /**
+     * Minimum number of frames a distance will have in the gradient.
```
Comment (ElDeveloper) [inline emperor/support_files/js/trajectory.js]: Worth adding the description in the comment above.

File: emperor/support_files/js/trajectory.js [hunk 3]
```diff
@@ -271,26 +278,26 @@ define([
...
+   * properties: x, y, z, name, color, P1, P2, P3, ... PN where N is the number
+   * of dimensions in this dataset.
+   * @param {string} trajectoryCategory a string with the name of the mapping
+   * file header where the data that groups the samples is contained, this will
+   * usually be BODY_SITE, HOST_SUBJECT_ID, etc..
+   * @param {string} gradientCategory a string with the name of the mapping file
    * header where the data that spreads the samples over a gradient is
    * contained, usually time or days_since_epoch. Note that this should be an
-   * all numeric category (required).
+   * all numeric category.
```
Comment (ElDeveloper) [inline emperor/support_files/js/trajectory.js]: Can you add an `@throw` section?

**Comment by ElDeveloper** [comment]
👍 just a couple minor comments.

**Comment by squirrelo** [comment]
Comments addressed, ready to go.

**Pull request merged**